controllers/scripts/common-env.sh

```diff
@@ -79,14 +84,25 @@ def gethost(data, host):
 
            if nodeExternalIP != '':
                externalIP = nodeExternalIP
+
+           labels = item['metadata']['labels']
+
+           if '${CONFIGURED_ACCESSIP_LABEL}' in labels:
+               configuredAccessIP = labels['${CONFIGURED_ACCESSIP_LABEL}']
+
+           if '${CONFIGURED_ALTERNATE_ACCESSIP_LABEL}' in labels:
+               configuredAlternateAccessIP = labels['${CONFIGURED_ALTERNATE_ACCESSIP_LABEL}']
+
            break
 
-    return internalIP + ' ' + externalIP
+    return internalIP + ' ' + externalIP + ' ' + configuredAccessIP + ' ' + configuredAlternateAccessIP
 
 print(gethost(data, host))")"
 
 export INTERNALIP=$(echo $HOSTIPS | awk '{print $1}')
 export EXTERNALIP=$(echo $HOSTIPS | awk '{print $2}')
+export CONFIGURED_ACCESSIP=$(echo $HOSTIPS | awk '{print $3}')
+export CONFIGURED_ALTERNATE_ACCESSIP=$(echo $HOSTIPS | awk '{print $4}')
```

Comment:
I guess this might cause issue when only `CONFIGURED_ALTERNATE_ACCESSIP_LABEL` label is present.
In that case,  `configuredAccessIP ` value be empty and `awk '{print $3}'` will actually give `configuredAlternateAccessIP` value and `awk '{print $4}'` will return empty value

Reply:
Fixed